stock_picking_accounting_date/models/stock_move.py

```diff
@@ -46,3 +46,10 @@ def _prepare_account_move_vals(
         if self.accounting_date:
             am_vals.update({"date": self.accounting_date})
         return am_vals
+
+    def _get_price_unit(self):
+        """Passes the accounting_date to be used in currency conversion for receipts
+        in foreign currency purchases."""
```

Comment:
```suggestion
        in foreign currency purchases.
        """
```

https://peps.python.org/pep-0257/

> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves. This way, Emacs’ fill-paragraph command can be used on it.